contracts/utils/LERC20.sol

```diff
@@ -214,6 +214,9 @@ contract LERC20 is Context, ILERC20 {
     }
 
     function _approve(address owner, address spender, uint256 amount) internal virtual {
+        require(owner != address(0), "ERC20: approve from the zero address");
```

Comment:
Is it really necessary?

Reply:
Here I decided to leave those required as is because that's the way they are on the ERC20 standard. One one issue it says we should remove them but on another issue it says that some other method is non ERC20 compliant. 

This check is not necessary. Shall I remove it directly?

Reply:
Ok, we can keep it.

Reply:
Got it, addressed on new commit 1ff03c9